modules/nf-core/pbsv/discover/main.nf

```diff
@@ -3,8 +3,8 @@ process PBSV_DISCOVER {
     label 'process_single'
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/pbsv:2.9.0--h9ee0642_0':
-        'biocontainers/pbsv:2.9.0--h9ee0642_0' }"
+        'oras://community.wave.seqera.io/library/pbsv:2.11.0--64a2437cb4119823':
```

Comment:
please use the https version of the singularity containers, as described here: https://nf-co.re/docs/tutorials/nf-core_components/using_seqera_containers